src/base/bittorrent/torrenthandle.h

```diff
@@ -54,14 +54,20 @@ namespace BitTorrent
         Unknown = -1,
 
         ForcedDownloading,
+        ForcedDownloadingNWTracker,
```

Comment:
Brevity is good but not at the expense of readability. If i read this outside the context of this PR i would have no idea what "Nw" means.

Reply:
@thalieht pretty sure it means "Not working"

Reply:
I've updated them.